cli/commands/create.go

```diff
@@ -60,6 +61,10 @@ func runCreateCommand(cmd *cobra.Command, args []string) error {
 		ctx.Create.Template = templates.CartridgeTemplateName
 	}
 
+	if ctx.Create.FileSystem == nil && ctx.Create.From == "" {
+		ctx.Create.FileSystem = static.CartridgeData
```

Comment:
In fact, `cartridge` template is just one of possible application templates (`create` command has `--template` flag, the default value is `cartridge`).
Sometimes we will add more templates (I believe).

So, generally, we choose Cartridge template filesystem only if `ctx.Create.Template` is `cartridge`. Otherwise, it should be an error.